src/pybind/mgr/test_orchestrator/module.py

```diff
@@ -20,41 +20,7 @@
 from mgr_module import MgrModule
 
 import orchestrator
```

Comment:
Remove this one

Reply:
that would blow-up the diff with unrelated changes.